bootcompiler/src/main/scala/org/mozartoz/bootcompiler/transform/DesugarClass.scala

```diff
@@ -299,7 +299,7 @@ object DesugarClass extends Transformer with TreeDSL {
     }
 
     atPos(method) {
-      PROC (name, Seq(selfParam, msgParam)) {
+      PROC (Some(Variable(new Symbol(name))), Seq(selfParam, msgParam)) {
```

Comment:
No `new Symbol` or it will create a new unique symbol. The Namer should take care of the transformation.
`None` is probably OK here in the meanwhile.